packages/form-handlers-mailchannels-test/src/index.ts

```diff
@@ -0,0 +1,33 @@
+import { mailchannelsHandler } from "@webstudio-is/form-handlers";
+
+export default {
+  async fetch(request: Request) {
+    const url = new URL(request.url);
+
+    const to = url.searchParams.get("to");
+
+    if (to == null) {
+      return new Response(
+        `Set "to" query parameter to your email. Example: https://...?to=bob@example.com`
```

Comment:
to to mb like:
Set the "to" query parameter with your email, like this: https://.../?to=bob@example.com

Reply:
This is just a package for developers to test, doesn't matter either way